llvm/test/Transforms/InstCombine/getelementptr.ll

```diff
@@ -111,6 +111,20 @@ define void @test_evaluate_gep_as_ptrs_array(ptr addrspace(2) %B) {
   ret void
 }
 
+define void @test_overaligned_vec(i8 %B) {
+        ; This should be turned into a constexpr instead of being an instruction
+; CHECK-LABEL: @test_overaligned_vec(
+; TODO: In this test case, half is overaligned to 32 bits.
+;       Vectors are bit-packed and don't respect alignment.
+;       Thus, the byte offset of the second half in <2 x half> is 2 bytes, not 4 bytes:
```

Comment:
Please do not interleave comments and CHECK lines. Move both comments outside the function.

Reply:
Thanks, I moved the first comment away. The TODO is already removed in the main commit of this PR.